FBC/FunctionalEntities/NorthAmericanEntities/USRegulatoryAgencies.rdf

```diff
@@ -628,6 +629,16 @@
 		<fibo-fnd-utl-av:explanatoryNote>EDGAR, the Electronic Data Gathering, Analysis, and Retrieval system, performs automated collection, validation, indexing, acceptance, and forwarding of submissions by companies and others who are required by law to file forms with the U.S. Securities and Exchange Commission (SEC). Its primary purpose is to increase the efficiency and fairness of the securities market for the benefit of investors, corporations, and the economy by accelerating the receipt, acceptance, dissemination, and analysis of time-sensitive corporate information filed with the agency.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:NamedIndividual>
 	
+	<owl:Class rdf:about="&fibo-fbc-fct-usjrga;EmployerIdentificationNumber">
+		<rdfs:subClassOf rdf:resource="&fibo-be-le-fbo;OrganizationIdentifier"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fct-usjrga;TaxpayerIdentificationNumber"/>
+		<rdfs:label>employer identification number</rdfs:label>
+		<skos:definition>unique nine-digit number assigned by the Internal Revenue Service (IRS) to business entities operating in the United States for the purposes of identification</skos:definition>
+		<fibo-fnd-utl-av:abbreviation>EIN</fibo-fnd-utl-av:abbreviation>
+		<fibo-fnd-utl-av:synonym>Federal Employer Identification Number (FEIN)</fibo-fnd-utl-av:synonym>
```

Comment:
"(FEIN)" should not be in the name but a separate acronym.

Reply:
done